troposphere/templates/allocations.html

```diff
@@ -14,14 +14,11 @@
         </div>
         <div class="message-wrapper">
 
-            <h1>Your account does not currently have a valid Allocation Source.</h1>
+            <h1>You do not currently have an <strong><em>ACTIVE</em></strong> Jetstream allocation.</h1>
 
-            <p>Your  will need a valid allocation in order to use Jetstream.</p>
-
-            <p>Jetstream allocations are done exclusively through the <a href="https://www.xsede.org/overview">eXtreme Science and Engineering Discovery Environment (XSEDE)</a>. You can read the <a href="https://www.xsede.org/getting-started">Getting Started guide</a> to learn about the process of getting onto XSEDE and applying for allocations and using XSEDE resources.</p>
-
-            <p>For more information on how to apply for an Allocation please visit the <a href="https://iujetstream.atlassian.net/wiki/display/JWT/Jetstream+Allocations">Jetstream Wiki</a>.</p>
+            <p>Jetstream allocations are awarded exclusively through the <a href="https://www.xsede.org/overview">eXtreme Science and Engineering Discovery Environment (XSEDE)</a>. For instructions about how to apply for an XSEDE portal account and an XSEDE allocation see <a href="https://iujetstream.atlassian.net/wiki/display/JWT/Jetstream+Allocations">https://iujetstream.atlassian.net/wiki/display/JWT/Jetstream+Allocations</a></p>
 
+            <p>If you have already applied for an allocation please allow 4-6 hours for processing</p>
```

Comment:
Right now, only Jetstream sees this. Because this is associated with `ALLOCATION_SOURCE` plugin, it is only activated in the Jetstream scenario. 

I want to have a notion of a "operating" or rendering context. So that we can offer up different verbiage for each, ideally loading a partial template from disk - or maybe having something set via SiteMetadata.